compiler/rustc_const_eval/src/const_eval/error.rs

```diff
@@ -79,8 +80,8 @@ pub fn get_span_and_frames<'tcx>(
     // Add notes to the backtrace. Don't print a single-line backtrace though.
     if stacktrace.len() > 1 {
         // Helper closure to print duplicated lines.
-        let mut add_frame = |mut frame: errors::FrameNote| {
-            frames.push(errors::FrameNote { times: 0, ..frame.clone() });
+        let mut add_frame = |mut frame: FrameNote| {
+            frames.push(FrameNote { times: 0, ..frame.clone() });
             // Don't print [... additional calls ...] if the number of lines is small
             if frame.times < 3 {
                 let times = frame.times;
```

Comment:
Is this logic still correct? It seems to be we could now have a situation where we see 2 frames that both have been seen already, so `last_frame.times` is 2, but these were actually 2 *different* frames and then this here will print the wrong thing.